tchannel/sync/client.py

```diff
@@ -103,7 +103,6 @@ def __init__(
 
         )
         self._threadloop = threadloop or ThreadLoop()
```

Comment:
Yeah, this also fixes a bug where the caller could have passed a started thread.